src/elements/content-sidebar/activity-feed/comment/__tests__/Replies.test.js

```diff
@@ -170,6 +172,27 @@ describe('elements/content-sidebar/ActivityFeed/comment/Replies', () => {
         expect(replyCreate).toBeCalledWith('Batman');
     });
 
+    test('should call onReplyDelete when reply is deleted', () => {
+        getWrapper({
+            replies: [comment, comment2],
+        });
+
+        const [menu] = screen.getAllByTestId('comment-actions-menu');
+
+        fireEvent.click(menu);
+
+        expect(screen.getByTestId('delete-comment')).toBeInTheDocument();
+        fireEvent.click(screen.getByTestId('delete-comment'));
+
+        expect(screen.getByTestId('bcs-delete-confirmation-delete')).toBeInTheDocument();
+        fireEvent.click(screen.getByTestId('bcs-delete-confirmation-delete'));
+
+        expect(replyDelete).toBeCalledTimes(1);
```

Comment:
Implementation detail; but I get that, at this level, it doesn't make sense to test `replyDelete` itself 👍

Reply:
Oh yeah, I just meant check anything to do with `replyDelete` being called. Ideally, we would assert what the effects of `replyDelete` being called are for the user. But it's super practical to do that here.